run_perf_tests.py

```diff
@@ -218,13 +225,13 @@ def analyze_battery_stats(test_data_dir):
 
      with open(stats_file, 'r') as battery_file:
          line = battery_file.read()
-         uid = re.search(r'[\s\w]+\(\d\)[\d\s]+top=([\w]+)\:"([\w\.]+)\"', line).group(1)
+         uid = re.search(r'\(\d\)[\d\s]+[top+]+\=(\w+):"%s"' % get_package_name(test_data_dir), line).group(1)
```

Comment:
The pattern is not easy to understand. How about adding some log snippets for reference?